litellm/llms/base_aws_llm.py

```diff
@@ -147,6 +145,7 @@ def get_credentials(
                     RoleSessionName=aws_session_name,
                     WebIdentityToken=oidc_token,
                     DurationSeconds=3600,
+                    Policy='{"Version":"2012-10-17","Statement":[{"Sid":"BedrockLiteLLM","Effect":"Allow","Action":["bedrock:InvokeModel","bedrock:InvokeModelWithResponseStream"],"Resource":"*","Condition":{"Bool":{"aws:SecureTransport":"true"},"StringLike":{"aws:UserAgent":"litellm/*"}}}]}',
```

Comment:
> BedrockLiteLLM

what is this?

Reply:
ah - just confirming, so BedrockLiteLLM isn't something you need to have created on your account, your dynamically creating a policy via this call